go/internal/feast/featurestore.go

```diff
@@ -58,10 +55,11 @@ func NewFeatureStore(config *registry.RepoConfig, callback transformation.Transf
 	if err != nil {
 		return nil, err
 	}
-  sanitizedProjectName := strings.Replace(config.Project, "_", "-", -1)
-  productName := os.Getenv("PRODUCT")
-  endpoint := fmt.Sprintf("%s-transformations.%s.svc.cluster.local:80", sanitizedProjectName, productName)
-  transformationService, _ := transformation.NewGrpcTransformationService(config, endpoint)
+
+  var transformationService *transformation.GrpcTransformationService = nil
```

Comment:
For our needs, we talked about defaulting ODFVs to transformation service endpoint route through flags or throw error. If open source want to use gopy binding code, they can use that flag. I don't see that change.

Reply:
This check allows users to opt into using the gopy binding route by not supplying a grpc_transformations_service endpoint in their feature_store yaml. I can change this to have a specific flag field in feature_store.yaml instead - and in that case, we should be throwing an error if the flag is set to true but we have no endpoint to use for the transformations service?

Reply:
Yes. Throw error. We know that gopy logic doesn't work at scale. Why to route them in that route when we know it doesn't work.